client/src/index.jsx

```diff
@@ -3,17 +3,116 @@ import ReactDOM from 'react-dom';
 
 import axios from 'axios';
 
+import RelatedItemList from './components/RelatedItemList.jsx';
+
+
 class App extends React.Component {
     constructor(props) {
         super(props);
+        this.state = {
+            relatedItems: [],
+            relatedItemInfo: [],
+            relatedItemsCurrent: [],
+            currentArr: 0,
+            maxRelatedLength: 5,
+            itemId: 40
+        }
+
+        // click bindings
+        this.arrowClick = this.arrowClick.bind(this);
```

Comment:
I never knew you could do this in the constructor of the class instead of passing the function and having to bind `this` right when it's passed to the class's child component.

Reply:
yup, I think this is the best practice place to have them.